src/Config/Loader/PhpFileLoader.php

```diff
@@ -52,7 +52,29 @@ public function load($file, $type = null)
             $code = substr($code, 0, -2);
         }
 
-        return rtrim($code)."\n";
+        // declare() statements
+        $lines = explode("\n", $code);
+        $codeNew = '';
+        $processed = false;
+
+        foreach ($lines as $line) {
+            if ('' === $line || $processed) {
```

Comment:
IMO this is wrong as it does not take file doc comments and lines with trailing whitespace into account.
IMO it will break on files with:
```php
<?php

/**
 * This file is part of foobar/baz.
 *
 * (c) 2017 Acme.
 * [...]
 */

declare(strict_types=1);

// [...]
```